packages/web-components/src/divider/divider.template.ts

```diff
@@ -0,0 +1,12 @@
+import { ElementViewTemplate, html } from '@microsoft/fast-element';
+import type { Divider } from './divider.js';
+
+/**
+ * Template for the Divider component
+ * @public
+ */
+export const template: ElementViewTemplate<Divider> = html<Divider>`
+  <template role="${x => x.role}" aria-orientation="${x => x.orientation}">
+    <slot></slot>
+  </template>
+`;
```

Comment:
We've aligned here, let's use the FAST template: https://github.com/microsoft/fast/blob/master/packages/web-components/fast-foundation/src/divider/divider.template.ts

Reply:
I don't understand what you want me to do here. The templates look exactly the same.

Reply:
```suggestion
export const template: ElementViewTemplate<Divider> = dividerTemplate();
```